development_scripts/demo_deconvolution.py

```diff
@@ -36,12 +29,25 @@
     data_folder / "O2_pulses.csv",
     reader="ixdat",
 )
+
 data_reg_cell_pulses_o2.plot_measurement()
-data_reg_cell_pulses_o2.set_siq_quantifier(
-    calibration=Calibration(cal_list=[F_o2]), carrier="He"
-)
 
-# prepare a figure to co-plot the measured and modelled data
+
+# ------------ O2 calibration
+# First, we mannually make a siq calibration point:
+F_o2 = MSCalResult(
+    name="O2", mol="O2", mass="M32", cal_type="ECMS calibration", F=0.11755
+).to_siq()
+# Then put it in a siq Calculator:
+calibration = plugins.siq.Calculator(cal_list=[F_o2])
+# And get the calculator ready to use on measurements:
+calibration.set_quantifier(mol_list=["O2"], mass_list=["M32"], carrier="He")
```

Comment:
It's because a `siqCalculator` can be used as a big library for all kinds of calibrations (need to check saving and loading!) and can also *predict* sensitivity factors, so the mol_list (likewise for mass_list) that makes sense to use when quantifying a certain experiment need not match the list of molecules for which the siqCalculator has sensitivity factors.
I could write code to automatically set the quantifier with the mol_list and mass_list matching what's in the siqCalculator if it's used without having explicitly set the quantifier, but this might be a bit of a mess for advanced quantification users (and simple quantification users can just stick to the EC-MS built-in!)

Reply:
Ah yes, this makes sense. I dont think it's necessary to do this automatically, but might help curious users to understand what is happening if this was documented somewhere. At least in the tutorial when taking the user through the steps, but ideally also in the demo and maybe even docstring?